docs/src/main/tut/faq.md

```diff
@@ -78,6 +79,24 @@ def even(i: Int): ErrorsOr[Int] = if (i % 2 == 0) i.validNel else s"$i is odd".i
 nl.traverse(even)
 ```
 
+## <a id="task" href="#task"></a>Where is IO/Task?
+
+In purely functional programming, a monadic `IO` or `Task` type is often used to handle side effects such as file/network IO. There [have](https://github.com/typelevel/cats/pull/894) [been](https://github.com/typelevel/cats/pull/907) [several](https://github.com/typelevel/cats/issues/1224) GitHub issues/PRs about this and many more Gitter/IRL conversations, but they all seem to arrive at the conclusion that there isn't a clear canonical `IO` or `Task` that serves everyones' needs. Some of the questions that come up are:
+
+- Should tasks be interruptible?
+- Should there be a single `Task` that subsumes `IO` and adds support for asynchrony and concurrency, or should there be separate `IO` and `Task` types?
+- How should concurrency be managed/configured?
+- Is [scala.js](https://www.scala-js.org/) supported?
+- Should you be able to block a thread waiting for the result of a task? This is really convenient for tests, but it isn't really compatible with a JavaScript runtime and therefore is an issue for [scala.js](https://www.scala-js.org/).
+
+For some use-cases, a very simple `IO` is the best answer, as it avoids a lot of the overhead and complexity of other solutions. However, other use-cases require low-level concurrency control with asynchrony, resource management, and stack-safety. Considering all of the competing concerns, Cats has opted to not implement its own `IO`/`Task` types and instead encourage users to use a separate library that best serves their use-case.
```

Comment:
an alternate approach is use `Free` and specify the things you need to do, then finally interpret those into `Xor`, `Try` or `Future` at the end of the world. I think this is a good approach too because generally the `IO` winds up being kind of an `Any` side-effect when most times you have a pretty narrow scope.

We already have some nice `Free` docs to point to, right?